en/middleware/mavlink.md

```diff
@@ -167,6 +167,18 @@ MavlinkReceiver::handle_message(mavlink_message_t *msg)
 		...
  	}
 ```
+
+## Alternative to Creating Custom MAVLink Messages
+Sometimes there is the need for custom MAVLink message whose content is not fully defined.
+
+For example when using MAVLink to interface PX4 with an embedded device, the messages that are exchanged between the autopilot and the device may go through several iterations before they are stabilized.
+In this case, it can be time-consumming and error-prone to regenerate the MAVLink headers, and make sure both devices use the same version of the protocol.
+
+An alternative - and temporary - solution is to repurpose [debug messages](../debug/debug_values.md). 
+Instead of creating a custom MAVLink message `CA_TRAJECTORY`, you can send a message `DEBUG_VECT` with the string key `CA_TRAJ` and data in the `x`, `y` and `z` fields.
+
+** Note: This solution is not efficient as it sends character string over the network and involves comparison of strings. It should be used for development only! **
```

Comment:
Format for notes should be as shown below (renders the content in a box in gitbook, looks OK in github.

```
> **Note** This solution is not efficient as it sends character string over the network and involves comparison of strings. It should be used for development only! 
```

Reply:
Thanks @hamishwillee I fixed it